### PR TITLE
Update card-tracker.mdx

### PR DESCRIPTION
**Pull request by mspohr**

### CODE REVIEW

File: docs/hardware/devices/seeed-studio/sensecap/card-tracker.mdx [hunk 1]
```diff
@@ -9,6 +9,8 @@ SenseCAP T1000-E is a high-performance tracker designed for Meshtastic. As small
 
 :::warning[Note]
 Currently, LR1110 radios are unable to receive Meshtastic packets from the older SX127x radios. Semtech is aware of this issue, and we're hopeful they'll provide a solution soon.
+Currently, this device is not supported in Meshtastic. There is no way to use the tracker functions.
```
Comment (tekstrand) [inline docs/hardware/devices/seeed-studio/sensecap/card-tracker.mdx:12]: Where did you find this information @mspohr? I have heard of many people using the T1000 successfully

**Comment by tekstrand** [comment]
@mspohr Garth's favorite pastime is closing GitHub pull requests and issues :) Don't take it too personally

Since you're looking for support to figure out how to accomplish something, I would suggest jumping onto the Discord server or posting on the subreddit, those seem to be the most active places for questions like this. PRs and issues to this repo are meant for changes to the documentation website only, it's not a way that the community provides support for questions. I'm sure the experienced folks in there will have ideas and solutions for what you're trying to do!

**Comment by mspohr** [comment]
Thanks for this information.
I have scoured through the Meshtastic web, Seeed web, Discord server (but
not Reddit) and have failed to find anyone using these Tracker devices as
trackers and haven't found any information on how I could possibly
configure them to track.
This frustration led me to post the warning on the Meshtastic wiki that
tracker functions are not supported.
I've also repeatedly requested that Seeed accept the return of the item for
a refund since I can't get them to do their intended function.
Seeed has been poorly responsive telling me that their "tech support" would
get back to me (they haven't in more than a week) and ignoring my request
for refund under their "30 day return policy".

.Mark

Mark Spohr, MD
***@***.***
*"Health is a state of complete physical, mental, and social well-being and
not merely the absence of disease or infirmity". (WHO)*

**Comment by garthvh** [comment]
There is a gps in the device which is why they have branded it as a tracker, what additional functionality do you expect from a tracker?

**Comment by mspohr** [comment]
I expect to be able to get data from the GPS.
No way to do that now.

Mark Spohr, MD
***@***.***
*"Health is a state of complete physical, mental, and social well-being and
not merely the absence of disease or infirmity". (WHO)*

**Comment by garthvh** [comment]
The gps works fine, the gps information is available in your phone app on the map like any other meshtastic node.

**Comment by mspohr** [comment]
Well, my phone has GPS so yes, I know where I am.
The problem is that I want to know where the tracker is... and there is no
way to query it.

Mark Spohr, MD
***@***.***
*"Health is a state of complete physical, mental, and social well-being and
not merely the absence of disease or infirmity". (WHO)*

**Pull request closed without merging**